test/Resolver/ArticleTeaserResolverTest.php

```diff
@@ -61,6 +88,58 @@ public function testGetDate(): void
         );
     }
 
+    public function testGetKicker(): void
+    {
+        $teaserA = $this->createArticleTeaser([
+            'base' => [
+                'teaser' => [
+                    'kicker' => 'Teaser-Kicker'
+                ],
+                'kicker' => 'Base-Kicker'
+            ]
+        ]);
+        $teaserB = $this->createArticleTeaser([
+            'base' => [
+                'kicker' => 'Base-Kicker'
+            ]
+        ]);
+        $teaserC = $this->createArticleTeaser([
+            'base' => [
+                'trees' => [
+                    'navigation' => [
+                        'parents' => [
+                            'parent' => [
+                                'id' => 'parent',
+                                'url' => '/parent.php'
+                            ]
+                        ]
+                    ]
+                ]
+            ]
+        ]);
+        $teaserD = $this->createArticleTeaser([]);
+        $this->assertEquals(
+            'Teaser-Kicker',
+            $this->resolver->getKicker($teaserA),
+            'unexpected teaser kicker'
+        );
+        $this->assertEquals(
+            'Base-Kicker',
+            $this->resolver->getKicker($teaserB),
+            'unexpected teaser kicker'
+        );
+        $this->assertEquals(
+            'Parent-Kicker',
+            $this->resolver->getKicker($teaserC),
+            'unexpected teaser kicker'
+        );
+        $this->assertEquals(
+            null,
+            $this->resolver->getKicker($teaserD),
+            'unexpected teaser kicker'
+        );
+    }
+
```

Comment:
That's three tests in one. They should be divided into 3 tests.

Reply:
Refactored it to have one assert per test only